tools/tertiary-analysis/dropletutils/dropletutils_macros.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<macros>
+
+  <token name="@TOOL_VERSION@">1.0.3</token>
+
+  <xml name="requirements">
+    <requirements>
+      <requirement type="package" version="0.0.1">dropletutils-scripts</requirement>
```

Comment:
I may have misunderstood, but are we not pinning to the underlying package version, rather than the scripts package version?